core/app/modules/site-editor/assets/js/templates/layout.js

```diff
@@ -7,7 +7,6 @@ import './site-editor.scss';
 export default function Layout( props ) {
 	const config = {
 		title: __( 'Theme Builder', 'elementor' ),
-		titleRedirectRoute: '/site-editor',
```

Comment:
This causes the THEME BUILDER "logo" to not be clickable when the Pro is active as well.
This needs to be overridden in the Pro. Find a way to make it not exist in the Core but Exist in the Pro.